### ISSUE TITLE
Fireworks don't make pollution

### ISSUE THREAD

**Issue by basdxz** [GTNewHorizons/GT-New-Horizons-Modpack#8029]
#### Which modpack version are you using?
Latest Zeta Test
#
#### If in multiplayer; On which server does this happen? 
Both.
#
#### What did you try to do, and what did you expect to happen?
I launched a firework, expected some pollution.
#
#### What happened instead? (Attach screenshots if needed)
https://cdn.discordapp.com/attachments/580258801371054092/852218347026382848/firework_pollution.mp4
#
#### What do you suggest instead/what changes do you propose?
@bartimaeusnek to fix it and make it create pollution like the Creeper Trophy so that I can have an immersive experience.
#
#### What is your GTNH Discord username?
Ven#6112 (I'm in the TecTech discord, it's cooler https://discord.gg/4EGHYvKDhH)

**Comment by Skiftcha** [comment]
so when you have achievement fireworks it will also make pollution?)

**Comment by basdxz** [comment]
yes because the firework effect is also server side (other players get to see it..)
